Automattic/wp-calypso
# Store Services: Allow unverifiable addresses to be bypassed

Fixes https://github.com/Automattic/woocommerce-services/issues/1326

This PR introduces a new "unverifiable" address view:

![screen shot 2018-05-24 at 10 45 38 am](https://user-images.githubusercontent.com/63922/40499833-5ca6871e-5f40-11e8-959c-a93a653a65b7.png)

**To test:**
* Enter a known "unverifiable" address _(See 1166378-zen)_
* Verify that new view shows
* Verify USPS and Google Maps links open new tabs/windows and go to sites _(Note: at present, the USPS verification app is down 😞 )_
* Verify that you can edit the address
* Verify that you can purchase a label with the address "as entered"

To do from review:

- [x] Fix left margin on address summary
- [x] Add media query to rearrange for narrow screens
- [x] Remove "suggestion" from `<AddressSummary>` class name
